src/parser/generated/regexp-tree.js

```diff
@@ -754,10 +754,33 @@ function getRange(text) {
  * Creates a character node.
  */
 function Char(value, kind, loc) {
+  let symbol;
+
+  switch (kind) {
+    case 'decimal': {
+      const code = parseInt(value.slice(1));
```

Comment:
nit: `Number` does parsing too, but a different one: the whole string should be parsed as number. The strictest for decimals is `parseInt(, 10)` (`10` is required to ban `0x`). `Number` allows `0[xob]` and floats.